webapp/src/components/admin_settings/jitsi_section.tsx

```diff
@@ -0,0 +1,323 @@
+import React from 'react';
+import {FormattedMessage} from 'react-intl';
+
+type Props = {
+    disabled: boolean,
+    onJitsiURLChange: (e: React.ChangeEvent<HTMLInputElement>) => void,
+    onJitsiEmbeddedChange: (e: React.ChangeEvent<HTMLInputElement>) => void,
+    onJitsiMeetingNamesChange: (e: React.ChangeEvent<HTMLInputElement>) => void,
+    onJitsiJwtAuthChange: (e: React.ChangeEvent<HTMLInputElement>) => void,
+    onJitsiAppIDChange: (e: React.ChangeEvent<HTMLInputElement>) => void,
+    onJitsiAppSecretChange: (e: React.ChangeEvent<HTMLInputElement>) => void,
+    onJitsiMeetingLinkExpChange: (e: React.ChangeEvent<HTMLInputElement>) => void,
+    onJitsiCompatibilityChange: (e: React.ChangeEvent<HTMLInputElement>) => void,
+    serverUrl: string,
+    embedded: boolean,
+    namingScheme: string,
+    jwtEnabled: boolean,
+    appID: string,
+    appSecret: string,
+    meetingLinkExpire: number,
+    compatibilityMode: boolean
+};
+
+export const JITSI_NAMING_SCHEME = {
+    WORDS: 'words',
+    UUID: 'uuid',
+    MATTERMOST: 'mattermost',
+    ASK: 'ask'
+};
+
+const JitsiSection = ({onJitsiAppIDChange, onJitsiEmbeddedChange, onJitsiMeetingLinkExpChange, onJitsiMeetingNamesChange, onJitsiJwtAuthChange, onJitsiAppSecretChange, onJitsiCompatibilityChange, onJitsiURLChange, serverUrl, embedded, namingScheme, jwtEnabled, appID, appSecret, meetingLinkExpire, compatibilityMode}: Props) => {
+    // TODO rewrite common components or use Mattermost components
```

Comment:
please fix this if possible
```suggestion
    // TODO: rewrite common components or use Mattermost components
```

Reply:
Fixed by Ayush in separate PR